docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md

```diff
@@ -0,0 +1,136 @@
+# Use Cascading Inputs in Tool
+
+Cascading settings between inputs are frequently used in situations where a selection in one input field determines what subsequent inputs should be shown. This approach helps create a more efficient, user-friendly, and error-free input process. This article will guide you through the process of implementing cascading settings for tool inputs.
+
+## Prerequisites
+Please ensure that your [Prompt flow for VS Code](https://marketplace.visualstudio.com/items?itemName=prompt-flow.prompt-flow) is updated to version 1.2.0 or later.
+
+
+## Create tool with cascading inputs
+We will provide a hands-on tool example to demonstrate the implementation of cascading settings between inputs within a tool. 
+In this tool, the availability of 'student_id' and 'teacher_id' inputs is determined by the value of the 'user_type' input. 
+Below shows how to support this cascading setting in both tool code and tool yaml.
+
+1. Develop your tool within the def function, referring to [tool_with_cascading_inputs.py](https://github.com/microsoft/promptflow/blob/main/examples/tools/tool-package-quickstart/my_tool_package/tools/tool_with_cascading_inputs.py) as an example. You need to pay attention to some key points:
```

Comment:
1. Develop the tool function, following the [cascading inputs example](https://github.com/microsoft/promptflow/blob/main/examples/tools/tool-package-quickstart/my_tool_package/tools/tool_with_cascading_inputs.py). Key points: